src/pages/contact_info.tsx

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import { Card, Image, Text, Badge, Button, Group, Grid } from '@mantine/core';
+
+const CustomCard = ({ title, operationHours, buttonText }) => (
+  <Card shadow="sm" padding="lg" radius="md" withBorder h={422} w={308}>
+    <div>
+      <Image
+        src="/assets/images/contact_info/Rectangle 1.png"
+        alt="Rectangle 1"
+        height={160}
+        p={25}
+        mb=""
+      />
+    </div>
+
+    <Group justify="space-between" mb="xs">
+      <Text fw={500}>{title}</Text>
+    </Group>
+
+    <Text size="sm" c="dimmed">
+      Operation Hours:
+    </Text>
+
+    <Text>{operationHours}</Text>
+
+    <Button variant="filled" color="blue" fullWidth mt="lg" radius="md" size="lg">
+      {buttonText}
+    </Button>
+  </Card>
+);
+
+
+
+
+// Export
+export default function contactInfoPage() {
+  return <div className="container mx-auto mt-12">
+    <section className="relative w-full">
+      {/* Header */}
+      <h1>
+      <div className="flex flex-col items-center">
+        <div className="text-5xl font-bold mb-4" >Contact Information</div>
+      </div>
+        </h1>
+
+        {/* Header.Description */}
+        <Text align="center" className="mb-16">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+</Text>
+
+ {/* Cards */}
+ <Grid align='center' style={{ margin: '50px 100px 50px 150px' }}>
+  {/* 1st Card */}
```

Comment:
These boilerplates `<Grid.Col span={4}>` can be improved by using `.map`

Here's one example:
```tsx
const contactInformationCards = [
{
    title: "Academic Support"
    operationHours: "Mon-Fri 08:30 - 17:30"
    buttonText: "Email"
},
{
    title: "Registra"
    operationHours: "Mon-Fri 08:30 - 17:30"
    buttonText: "Email"
},
{
    title: "Library"
    operationHours: "Mon-Fri 08:30 - 17:30"
    buttonText: "Email"
},
.... // more contact information
]

// when using it
<Group position="center">
	{contactInformationCards.map((card) => (
		<CustomCard key={"contactInformationCards" + card.title} {...card} />
	))}
</Group>

```

Just some idea though, I took it from:
https://github.com/stamford-syntax-club/stamfordcenter-frontend/blob/main/src/components/page_sections/QuickLinksSection.tsx#L129C7-L129C7

This way it is easier to add more cards in the future (especially when we integrate with backend and the contents will now be dynamic)